### PR TITLE
FixingSomeFeatures 

### PR DESCRIPTION
**Pull request by believer-0910**

### CODE REVIEW

File: src/main/java/com/exadel/hotel/room/controller/RoomController.java [hunk 1]
```diff
@@ -32,21 +33,23 @@ public ResponseEntity<List<RoomDto>> getRoomsByFloorId(@PathVariable("floorId")
...
     @PostMapping("/add")
     public ResponseEntity<RoomDto> add(@RequestBody RoomDto roomDto) {
-        return ResponseEntity.ok(roomService.add(roomDto));
+        return new ResponseEntity<>(roomService.add(roomDto), HttpStatus.CREATED);
     }
 
     @PutMapping("/update/{id}")
     public ResponseEntity<RoomDto> update(@PathVariable("id") Long id, @RequestBody RoomDto roomDto) {
-        return ResponseEntity.ok(roomService.update(id, roomDto));
+        return new ResponseEntity<>(roomService.update(id, roomDto), HttpStatus.ACCEPTED);
```
Comment (KirvelPasha) [inline src/main/java/com/exadel/hotel/room/controller/RoomController.java]: I don't think ACCEPTED it's correct  you could check here: https://stackoverflow.com/questions/2342579/http-status-code-for-update-and-delete 
I think  here you can return 201

File: src/main/java/com/exadel/hotel/type/controller/TypeController.java [hunk 2]
```diff
@@ -26,21 +27,23 @@ public ResponseEntity<TypeDto> get(@PathVariable("id") Long id) {
...
 
     @PutMapping("/update/{id}")
     public ResponseEntity<TypeDto> update(@PathVariable("id") Long id, @RequestBody TypeDto typeDto) {
-        return ResponseEntity.ok(typeService.update(id, typeDto));
+        return new ResponseEntity<>(typeService.update(id, typeDto), HttpStatus.ACCEPTED);
     }
 
     @DeleteMapping("/delete/{id}")
-    public void delete(@PathVariable("id") Long id) {
+    public ResponseEntity<Boolean> delete(@PathVariable("id") Long id) {
```
Comment (KirvelPasha) [inline src/main/java/com/exadel/hotel/type/controller/TypeController.java]: It's a bit strange, you don't send response body.
Let's go to do
 public ResponseEntity<Void> delete(@PathVariable("id") Long id) {
        typeService.delete(id);
        return new ResponseEntity<>(HttpStatus.OK);
    }

Comment (believer-0910) [reply src/main/java/com/exadel/hotel/type/controller/TypeController.java]: in this case sonarlint give me advise that I should add ResponseEntity type

Comment (believer-0910) [reply src/main/java/com/exadel/hotel/type/controller/TypeController.java]: Or if it is possibe and nothing to worth I will change it

**Comment by KirvelPasha** [comment]
Please review your  @RequestMapping e.g. @RequestMapping("/room") i think it's better to use @RequestMapping("/rooms")

**Pull request merged**